autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -1,6 +1,118 @@
+import asyncio
+import time
+from typing import List
 from django.core.management import base
+from scraper.banner_requests import BannerRequests
+from scraper.models import Course, Instructor, Section, Meeting
+from course import generate_course_id
+
+def parse_section(course):
+    #gets values to create object
+    section_id = ''
+    section_subject = course['subject']
+    course_number = course['courseNumber']
+    section_number = ''
+    section_term_code = course['term']
+    
+    section_min_credits = course['creditHourLow']
+    section_max_credits = course['creditHourHigh']
+
+    section_max_enrollment = course['maximumEnrollment']
+    section_current_enrollment = course['enrollment']
+    section_instructor = ''
+    for i in course['faculty']:
+        instructor_name = i['displayName']
+        section_instructor += instructor_name + ','
+
+    #creates and saves section object
+    s = Section(id=section_id, subject=section_subject, course_num=course_number, section_num=section_number, term_code=section_term_code, min_credits=section_min_credits, max_credits=section_max_credits, max_enrollment=section_max_enrollment, current_enrollment=section_current_enrollment, instructor=section_instructor)
```

Comment:
While we may be able to pass in the primary key for the Instructor, instead I think `parse_instructor` should return an `Instructor` model (that's already saved to the DB), which you can then pass into as an argument to `parse_section`. Then when you create the Section object, you simply set the `instructor` field to the `Instructor` model that was passed into the function.